website/source/api/catalog.html.md

```diff
@@ -399,7 +399,8 @@ The table below shows this endpoint's support for
   the datacenter of the agent being queried. This is specified as part of the
   URL as a query parameter.
 
-- `tag` `(string: "")` - Specifies the tag to filter on.
+- `tag` `(string: "")` - Specifies the tag to filter on. This is specified as part of 
+  the URL as a query parameter. Can be used multiple times for additional filtering.
```

Comment:
It's pretty obvious to me from the way you worded it but is it worth being even more explicit that when multiple tags are given they must ALL match?

Reply:
I've added some additional clarification both places this shows up.